src/main/java/htsjdk/io/Writer.java

```diff
@@ -0,0 +1,20 @@
+package htsjdk.io;
+
+import java.io.Closeable;
+
+public interface Writer<A> extends Closeable {
```

Comment:
I agree, I just wanted to minimize changes for this PR.

Reply:
If it's a couple of minutes work, let's maybe add this to the SAM filter writer types and the VariantContextWriter since those are the two other big ones?